<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>995</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>961</width>
      <height>651</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Orden</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>341</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string>Datos</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>151</width>
         <height>351</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Cliente</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Empleado</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Agencia</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Peso</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Nombre del Barco</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Direccion de Envio</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Ciudad</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Region</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Codigo postal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Pais</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>151</width>
         <height>341</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="CB_Clientes"/>
        </item>
        <item>
         <widget class="QComboBox" name="CB_Empleado"/>
        </item>
        <item>
         <widget class="QComboBox" name="CB_Agencia"/>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_Peso"/>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_NombreBarco"/>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_DireccionEnvio"/>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_CiudadEnvio"/>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_RegionEnvio"/>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_CodigoPostal"/>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_Pais"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="PB_CrearOrden">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>380</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Crear</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="emitirorden">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>90</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Emitir Orden </string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>411</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Agregar detalle</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>151</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Producto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Cantidad</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Descuento</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>221</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QComboBox" name="CB_Productos"/>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_cantidad"/>
        </item>
        <item>
         <widget class="QLineEdit" name="LE_descuento"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="PB_agregardetalles">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>121</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Agregar</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>220</y>
        <width>541</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Detalles</string>
      </property>
      <widget class="QTableView" name="TV_detalles">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>521</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Detalles de la Orden</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Productos</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Generar Reportes</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>521</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Reportes</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>471</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QPushButton" name="PB_reporteCliente">
          <property name="text">
           <string>Generar reporte para cliente</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_reporteProveedor">
          <property name="text">
           <string>Generar reporte para proveedor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_reporteCategoria">
          <property name="text">
           <string>Generar reporte para categoria</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
